config/redirects

```diff
@@ -6,4 +6,5 @@ raw: ${prefix}/ -> ${base}/current/
 raw: ${prefix}/master -> ${base}/upcoming/
 
 # Redirects for Analyzer Rules
-[*-master]: ${prefix}/${version}/rules/ -> ${base}/${version}/
\ No newline at end of file
+[*-master]: ${prefix}/${version}/rules/ -> ${base}/${version}/
+raw: ${prefix}/current/rules/ -> ${base}/current/
```

Comment:
Question:
Would there ever be a good reason to link to "current" from code or release notes, or should it be pegged to a specific version? It seems strange to link to "current/rules" for release notes that refer to a specific version since "current" will change.

Reply:
The rules page that the links in the nuget package will link to is going to be a list of all rules (regardless of the version they were released in) with explanations of each one. So they wanted the links to go to the most up do date page describing the rules (even though they're unlikely to change over versions).
 
In the latest package release (v1.3) the links they added all go to "current" and none are tied to a specific version at this time.

Reply:
Ok, so the "rules/" page will list all the rules on a single page? So once that page exists, this redirect and the previous one should be removed, correct? If there's a JIRA ticket that explains the task, could you provide the link in the description or send it to me? Thanks!

Reply:
Right. Here's the Jira ticket for the implementation of rules/removal of these redirects: https://jira.mongodb.org/browse/DOCSP-32811